python/ray/tune/tests/test_integration_pytorch_lightning.py

```diff
@@ -8,13 +8,18 @@
 
 from torch.utils.data import DataLoader, Dataset
 
+from importlib_metadata import version
+from packaging.version import parse as v_parse
+
 from ray import tune
 from ray.tune.integration.pytorch_lightning import (
     TuneReportCallback,
     TuneReportCheckpointCallback,
     _TuneCheckpointCallback,
 )
 
+ray_pl_use_master = v_parse(version("pytorch_lightning")) >= v_parse("1.6")
```

Comment:
Looks like we may need the same changes in https://github.com/ray-project/ray/blob/master/python/ray/tests/ray_lightning/simple_tune.py